packages/components/src/styles/container/container.css

```diff
@@ -20,4 +20,54 @@
   &--padding-sm {
     @apply px-6 py-4;
   }
+
+  --triangle-background: white;
+
+  &--triangle-top::before {
+    @apply absolute block left-[calc(50%-15px)] top-0 content-[''] border-solid border-transparent border-[15px] border-t-[var(--triangle-background)];
+  }
+
+  &--triangle-right::before {
+    @apply absolute block top-[calc(50%-15px)] right-0 content-[''] border-solid border-transparent border-[15px] border-r-[var(--triangle-background)];
+  }
+
+  &--triangle-bottom::before {
+    @apply absolute block left-[calc(50%-15px)] bottom-0 content-[''] border-solid border-transparent border-[15px] border-b-[var(--triangle-background)];
+  }
+
+  &--triangle-left::before {
+    @apply absolute block top-[calc(50%-15px)] left-0 content-[''] border-solid border-transparent border-[15px] border-l-[var(--triangle-background)];
+  }
+
+  &--triangle-top-border::after {
+    @apply absolute block left-[calc(50%-15px)] -top-[1px] content-[''] border-transparent border-solid border-[15px] border-t-[var(--triangle-background)];
+  }
+
+  &--triangle-top-border::before {
+    @apply absolute block left-[calc(50%-16px)] -top-[1px] content-[''] border-transparent border-solid border-[16px] border-t-neutral-400;
+  }
+
+  &--triangle-right-border::after {
+    @apply absolute block top-[calc(50%-15px)] -right-[1px] content-[''] border-transparent border-solid border-[15px] border-r-[var(--triangle-background)];
+  }
+
+  &--triangle-right-border::before {
+    @apply absolute block top-[calc(50%-16px)] -right-[1px] content-[''] border-transparent border-solid border-[16px] border-r-neutral-400;
+  }
+
```

Comment:
Just curious why is the code structured like that:
```
&--triangle-right-border::after {
  @apply absolute block top-[calc(50%-15px)] -right-[1px] content-[''] border-transparent border-solid border-[15px] border-r-[var(--triangle-background)];
}

&--triangle-right-border::before {
  @apply absolute block top-[calc(50%-16px)] -right-[1px] content-[''] border-transparent border-solid border-[16px] border-r-neutral-400;
}
```
and not like (not sure if it works, I just edited it in here ;) ):
```
&--triangle-right-border{
  &&after, &&before {
    @apply absolute block -right-[1px] content-[''] border-transparent border-solid;
  }

  &&before {
    @apply top-[calc(50%-16px)] border-[16px] border-r-neutral-400;
  }

  &&after {
    @apply top-[calc(50%-15px)] border-[15px] border-r-[var(--triangle-background)];
  }
}
```

Reply:
I guess because of the the file is ".css" I didn't think to use nested selectors despite the obvious level one which was there. I have adjusted accordingly.